lines.js

```diff
@@ -52,6 +52,9 @@ function startDraw(e) {
             projectHistory.splice(0, currentIndex);
             currentIndex = 0;
         }
+        if (historyForward.classList.contains("activated") === true) {
```

Comment:
`Array.prototype.contains` returns `Boolean` already, so `=== true` is not necessary